yt_dlp/extractor/rheinmaintv.py

```diff
@@ -0,0 +1,100 @@
+    }, {
+        'url': 'https://www.rheinmaintv.de/sendungen/beitrag-video/casino-mainz-bei-den-deutschen-meisterschaften/vom-14.11.2022/',
+        'info_dict': {
+            'id': 'casino-mainz-bei-den-deutschen-meisterschaften-vom-14.11.2022',
+            'ext': 'ismv',
+            'title': 'Casino Mainz bei den Deutschen Meisterschaften',
+            'view_count': int,
+            'timestamp': 1668527402,
+            'alt_title': 'Casino Mainz bei den Deutschen Meisterschaften',
+            'upload_date': '20221115',
+            'display_id': 'casino-mainz-bei-den-deutschen-meisterschaften',
+            'duration': 348.0,
+            'thumbnail': r're:^https://.+\.jpg',
+            'description': 'md5:70fc1660eeba96da17199e5bdff4c0aa',
+        },
+        'params': {'skip_download': 'ism'},
+    }, {
+        'url': 'https://www.rheinmaintv.de/sendungen/beitrag-video/bricks4kids/vom-22.06.2022/',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        mobj = self._match_valid_url(url)
+        display_id = mobj.group('display_id')
+        video_id = mobj.group('video_id').replace('/', '-')
+        webpage = self._download_webpage(url, video_id)
+
+        source, img = self._search_regex(r'(?s)(?P<source><source[^>]*>)(?P<img><img[^>]*>)',
+                                         webpage, 'video', group=('source', 'img'))
+        source = extract_attributes(source)
+        img = extract_attributes(img)
+
+        # Work around the method self._json_ld (called by self._search_json_ld), which
+        # extracts the useless 'contentUrl' (as 'url') instead of the essential 'embedUrl'.
+        raw_json_ld = list(self._yield_json_ld(webpage, video_id))
+        json_ld = self._json_ld(raw_json_ld, video_id)
+
+        ism_manifest_url = (
+            source.get('src')
+            or next(json_ld.get('embedUrl') for json_ld in raw_json_ld if json_ld.get('@type') == 'VideoObject')
+        )
+        formats, subtitles = self._extract_ism_formats_and_subtitles(ism_manifest_url, video_id)
+
+        return {
+            'id': video_id,
+            'display_id': display_id,
+            'title':
+                self._html_search_regex(r'<h1><span class="title">([^<]*)</span>',
+                                        webpage, 'headline', default=None)
+                or img.get('title') or json_ld.get('title') or self._og_search_title(webpage)
+                or remove_end(self._html_extract_title(webpage), ' -'),
+            'alt_title': img.get('alt'),
+            'description': json_ld.get('description') or self._og_search_description(webpage),
+            'formats': formats,
+            'subtitles': subtitles,
+            'thumbnails': [{'url': img['src']}] if 'src' in img else json_ld.get('thumbnails'),
+            **traverse_obj(json_ld, {
+                'timestamp': 'timestamp',
+                'duration': 'duration',
+                'view_count': 'view_count',
+            }),
```

Comment:
Why not merge_dict with the whole thing?

Reply:
I think the original PR author wanted to mix and match, they wanted to give priority to JSON LD for certain fields but also wanted non-JSON LD fallbacks for those (`description`, `title`)

But yeah ig merging the whole thing with JSON LD as secondary dict couldn't hurt